esp8266-fastled-webserver/esp8266-fastled-webserver.ino

```diff
@@ -311,22 +424,34 @@ void setup() {
   httpUpdateServer.setup(&webServer);
 
   webServer.on("/all", HTTP_GET, []() {
-    String json = getFieldsJson(fields, fieldCount);
+    String json = getFieldsJson();
+    webServer.sendHeader("Access-Control-Allow-Origin", "*");
+    webServer.send(200, "application/json", json);
+  });
+  
+  webServer.on("/product", HTTP_GET, []() {
+    String json = "{\"productName\":\"" PRODUCT_FRIENDLY_NAME "\"}";
+    webServer.sendHeader("Access-Control-Allow-Origin", "*");
+    webServer.send(200, "application/json", json);
+  });
+
+  webServer.on("/info", HTTP_GET, []() {
+    String json = getInfoJson();
     webServer.sendHeader("Access-Control-Allow-Origin", "*");
     webServer.send(200, "application/json", json);
   });
 
   webServer.on("/fieldValue", HTTP_GET, []() {
     String name = webServer.arg("name");
-    String value = getFieldValue(name, fields, fieldCount);
+    String value = getFieldValue(name);
```

Comment:
Cleaner API, no exposure to the internal details of how the fields are stored need to be exposed here.